### ISSUE TITLE
对70B base model和14B base model进行evaluate 72B loss的值 大于 14B loss

### ISSUE THREAD

**Issue by wccccp** [QwenLM/Qwen#926]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

使用hugging face对相同的数据集，对模型 forward 算loss  得出的结果72b base loss 大于 14B base loss  请问是出了什么问题 如何解决？ 
72B:{
    "eval_loss": 4.2626953125,
    "eval_runtime": 129.083,
    "eval_samples_per_second": 0.248,
    "eval_steps_per_second": 0.124
}
14B:{
    "eval_loss": 2.5419921875,
    "eval_runtime": 31.2478,
    "eval_samples_per_second": 2.048,
    "eval_steps_per_second": 0.512
}

### 运行环境 | Environment

```Markdown
- OS:
- Python: 3.8
- Transformers: 4.33.0
- PyTorch: 2.0.1
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):11.7
```

**Comment by jklj077** [comment]
请提供复现方法（测试用的脚本）。

**Comment by wccccp** [comment]
> 请提供复现方法（测试用的脚本）。

您好 发您邮件了 麻烦看一下

**Comment by jklj077** [comment]
经确认，我们(qianwen_opensource@alibabacloud.com)没有收到您发送的邮件。另外，建议提供公开的最小可复现脚本。

**Comment by wccccp** [comment]
> 经确认，我们([qianwen_opensource@alibabacloud.com](mailto:qianwen_opensource@alibabacloud.com))没有收到您发送的邮件。另外，建议提供公开的最小可复现脚本。

sry 发您个人邮件了 已经重新发了官方邮件
